deepchecks/base/display_pandas.py

```diff
@@ -53,14 +53,30 @@ def dataframe_to_html(df: Union[pd.DataFrame, Styler]):
         return df.to_html()
 
 
-def display_conditions_table(conditions_table: List):
+def display_conditions_table(check_results: Union['CheckResult', List['CheckResult']]):
     """Display the conditions table as DataFrame.
 
     Args:
-        conditions_table (List): list that contains the conditions in table format
+        check_results (Union['CheckResult', List['CheckResult']]): check results to show conditions of.
     """
-    conditions_table = pd.DataFrame(data=conditions_table,
-                                    columns=['Status', 'Check', 'Condition', 'More Info', 'sort'], )
+    if not isinstance(check_results, List):
+        show_check = False
+        check_results = [check_results]
+    else:
+        show_check = True
+
+    table = []
+    for check_result in check_results:
+        for cond_result in check_result.conditions_results:
+            sort_value = cond_result.get_sort_value()
+            icon = cond_result.get_icon()
+            header = check_result.get_header()
+            table.append([icon, header, cond_result.name, cond_result.details, sort_value])
+
+    conditions_table = pd.DataFrame(data=table,
+                                    columns=['Status', 'Check', 'Condition', 'More Info', 'sort'])
     conditions_table.sort_values(by=['sort'], inplace=True)
     conditions_table.drop('sort', axis=1, inplace=True)
+    if show_check is False:
```

Comment:
what does it do?

Reply:
what do you mean? the specific line checks if the parameter is False

Reply:
I got it now, so you won't show the name again